Model/Hero.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Model;
+
+class Hero
+{
+    private $id;
+    private $name;
+    private $jediFactor;
+    private $team;
+
```

Comment:
I think there should be the static variables for the teams here. And then have a `getValidTeams` method that will return them in an array like we did with ships. This will help with validation on your forms.

Reply:
Added statics and updated calls